lab_1/main.py

```diff
@@ -51,7 +92,21 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+    try:
```

Comment:
why try? ifs should do the job

Reply:
Because it seemed easier to resolve detected mistakes through try and except. If needed, I'll rewrite them.

Reply:
Okay I see your comments about excepts, will be done tomorrow.

Reply:
try/except is not the best option if we can write a simple `if`. There is no point